app/src/main/java/com/alexandruvalentinconstantin/newsreader/feature/newslist/adapter/RecyclerViewAdapter.java

```diff
@@ -0,0 +1,61 @@
+package com.alexandruvalentinconstantin.newsreader.feature.newslist.adapter;
+
+import android.annotation.SuppressLint;
+import android.view.LayoutInflater;
+import android.view.ViewGroup;
+
+import androidx.annotation.NonNull;
+import androidx.recyclerview.widget.RecyclerView;
+
+import com.alexandruvalentinconstantin.newsreader.databinding.ArticleNewItemBinding;
+import com.alexandruvalentinconstantin.newsreader.feature.newslist.listener.OnArticleSelected;
+import com.alexandruvalentinconstantin.newsreader.feature.newslist.model.ArticleItemViewModel;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class RecyclerViewAdapter extends RecyclerView.Adapter<RecyclerViewAdapter.TaskViewHolder> {
```

Comment:
RecyclerViewAdapter is named it could serve all recycler but it's not that general, let's rename this class to something more specific